Projects/UOContent/Items/Containers/Fillable Containers/FillableContainers.cs

```diff
@@ -7,21 +7,8 @@ namespace Server.Items;
 public partial class LibraryBookcase : FillableContainer
 {
     [Constructible]
-    public LibraryBookcase() : base(0xA97) => Weight = 1.0;
-
-    public override bool IsLockable => false;
-    public override int SpawnThreshold => 5;
-
-    protected override int GetSpawnCount() => 5 - GetItemsCount();
-
-    public override void AcquireContent()
+    public LibraryBookcase() : base(0xA97)
     {
-        if (ContentType != FillableContentType.None)
-        {
-            return;
-        }
-
-        ContentType = FillableContentType.Library;
```

Comment:
Was there something wrong with this? 🤔 - The `ContentType` change will cause a respawn, which is what we want right?